.github/workflows/ci.yml

```diff
@@ -11,8 +11,8 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest, macos-latest, windows-latest]
-        # At the time of writing this pypy3.10 is not available through setup-python
-        python-version: [3.8, 3.9, "3.10", pypy3.8, pypy3.9]
+        # At the time of writing this pypy3.11/3.12 is not available through setup-python
+        python-version: ["3.8", "3.9", "3.10", "3.11", "3.12", pypy3.8, pypy3.9, pypy3.10]
```

Comment:
Please limit the PR to changes related to the title (quote changes and PyPy 3.10 are out of scope)

Reply:
Ok, I've removed the quotes and pypy3.10. Also dropped the comment change, as it is kinda weird if it would say "pypy3.11/3.12 is not available through setup-python", while still not using pypy3.10 :sweat_smile: .

I'll create a follow-up PR to add pypy3.10 once this is merged.

Reply:
Cheers